README.md

```diff
@@ -78,18 +78,19 @@ It makes use of,
 Creating the files below in your project directory:
 
 ```sh
-├── docker-compose.kev.stage.yaml       # stage Compose environment override file
+├── docker-compose.kev.stage.yaml       # dev sandbox Compose environment override file
```

Comment:
this should be `docker-compose.kev.dev.yaml` I guess?

Reply:
Oops, will fix.